snuba/datasets/configuration/generic_metrics/entities/gauges.yaml

```diff
@@ -130,6 +137,13 @@ storages:
   - storage: generic_metrics_gauges_raw
     is_writable: true
     translation_mappers:
+      columns:
+        - mapper: ColumnToColumn
+          args:
+            from_table_name:
+            from_col_name: timestamp
+            to_table_name:
+            to_col_name: rounded_timestamp
```

Comment:
For `generic_metrics_gauges_raw`, it looks like the column name is `timestamp`. Do we still need this mapper?